packages/hoppscotch-sh-admin/locales/af.json

```diff
@@ -0,0 +1,132 @@
+{
+  "app": {
+    "collapse_sidebar": "",
+    "expand_sidebar": "",
+    "name": "",
+    "no_name": "",
+    "open_navigation": ""
+  },
+  "metrics": {
+    "dashboard": "",
+    "no_metrics": "",
+    "total_collections": "",
+    "total_requests": "",
+    "total_teams": "",
+    "total_users": ""
+  },
+  "role": {
+    "editor": "",
+    "owner": "",
+    "viewer": ""
+  },
```

Comment:
Keep these files empty. Just put empty JSON object here `{}`. As we don't have any translation given in these files we should keep it empty. Since empty string is also a value.